specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/examples/SearchServiceCreateAlias.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
```

Comment:
Is there missing update for alias count in the service stats example?

Reply:
Good catch, thanks!